.github/workflows/integrate.yaml

```diff
@@ -0,0 +1,57 @@
+
+name: Continuous Integration
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+    branches:
+      - main
+
+jobs:
+    Lint:
+      runs-on: ubuntu-latest
+
+      steps:
+      - name: Check out repository code
+        uses: actions/checkout@v2
+
+      - name: Set up Python 3.9
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.9
+
+      - name: Install pipenv and all dependencies for Yuptoo
+        run: |
+            pip install pipenv
+            pipenv install --dev
+
+      - name: Lint with flake8
+        run: |
+            pip install flake8
+            flake8 .
```

Comment:
@r14chandra can we directly run flake8 using pipenv `pipenv run flake8` instead of separately installing it. We have flake8 in our dev dependence and as you are performing `pipenv install --dev ` it should install flake8 at that time.

Reply:
Yeah I didn't see the packages under `pipfile` earlier. Fixed it! Thanks